tests/test_lint_recipe.py

```diff
@@ -266,6 +266,23 @@ def test_jinja_os_environ(self):
                          """)
             lints = linter.main(recipe_dir)
 
+    def test_jinja_load_file_regex(self):
+        # Test that we can use load_file_regex in a recipe. We don't care about
+        # the results here.
+        with tmp_directory() as recipe_dir:
+            with io.open(os.path.join(recipe_dir, 'sha256'), 'w') as fh:
+                fh.write("""
+                        fake  fakefile.txt
+                        """)
+            with io.open(os.path.join(recipe_dir, 'meta.yaml'), 'w') as fh:
+                fh.write("""
+                        {% set sha256 = load_file_regex('shasum', '.*') %}
+                        package:
+                          name: foo
+                          version: {{ version }}
+                        """)
+            lints = linter.main(recipe_dir)
```

Comment:
What is this testing?

Reply:
it tests that if a recipe calls "load_file_regex" the linter does not choke as seen in the issues linked to this PR

Reply:
Got it. Thanks for the exaplanation